dataservice/api/diagnosis/models.py

```diff
@@ -32,6 +34,10 @@ class Diagnosis(db.Model, Base):
     age_at_event_days = db.Column(db.Integer(),
                                   doc='Age at the time of event occurred in '
                                       'number of days since birth.')
+    mondo_id = db.Column(db.Text(),
+                         doc='An ontology that harmonizes diagnosis')
```

Comment:
Maybe change the docs to: 
- `The ID of the term from the Monary Disease Ontology which represents a harmonized diagnosis`.
- `The ID of the term from the International Classification of Diseases which represents a harmonized diagnosis.`

Reply:
👍